mythtv/programs/scripts/metadata/Television/ttvdb4_doctests

```diff
@@ -0,0 +1,563 @@
+# -*- coding: UTF-8 -*-
+
+
+# Doc-Tests for ttvdb4
+# file: ttvdb4_doctests
+
+# Note: These tests take several minutes to finish! Please be patient.
+
+# ttvdb4.py -l en -C 360893
+>>> sys.argv = shlex.split('ttvdb4 -l en -C 360893')
+>>> main()
+<?xml version='1.0' encoding='UTF-8'?>
+<metadata>
+  <item>
+    <title>Chernobyl</title>
+    <description>Chernobyl dramatizes the story of the 1986 nuclear accident — one of the worst man-made catastrophes in history — and the sacrifices made to save Europe from unimaginable disaster.</description>
+    <inetref>360893</inetref>
+    <homepage>https://www.hbo.com/chernobyl/season-1</homepage>
+    <language>en</language>
+    <userrating>8.700000</userrating>
```

Comment:
consider wildcarding anything that could be brittle. e.g. userrating is something that could change